### PR TITLE
Support rust

### PR DESCRIPTION
**Pull request by HongjiangHuang**
I have generated and tested this https://github.com/HongjiangHuang/kube-rust repository.

### CODE REVIEW

File: openapi/rust.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <skipValidateSpec>true</skipValidateSpec>
+                            <inputSpec>${generator.spec.path}</inputSpec>
+                            <generatorName>rust</generatorName>
+                            <gitUserId>kubernetes-rust</gitUserId>
```
Comment (brendandburns) [inline openapi/rust.xml:23]: kubernetes-client

**Comment by brendandburns** [comment]
This looks good to me, do you have a repo somewhere that has the generated code?

**Comment by HongjiangHuang** [comment]
I have generated and tested this https://github.com/HongjiangHuang/kube-rust repository.
Now you can wait for this [PR](https://github.com/OpenAPITools/openapi-generator/pull/9745) to merge or `export OPENAPI_GENERATOR_COMMIT=381af95daf00055dbc1cffda9e17c6e90e95f886`

**Comment by brendandburns** [comment]
/lgtm

Once that generator fix is merged, I will approve this.

Once it is approved, the next step is to apply for a new repo in https://github.com/kubernetes/org 

Here's the one that added 'C' for reference:

https://github.com/kubernetes/org/issues/1700

We will need two maintainers for the library, I assume you are interested in being one of them. I will sign up to be the other one.

**Comment by HongjiangHuang** [comment]
> We will need two maintainers for the library, I assume you are interested in being one of them. I will sign up to be the other one.

I am very happy to, Thank you

**Comment by brendandburns** [comment]
Great. Please ping this PR again when the openapi-client fix merges and we can merge this PR and then move on to officially adding the repo.

**Comment by HongjiangHuang** [comment]
Hi, @brendandburns

Now the pr https://github.com/OpenAPITools/openapi-generator/pull/9745 has been merged.

**Comment by zmrush** [comment]
i'm interested in it

**Comment by brendandburns** [comment]
/approve

The next step is to create the appropriate repo (https://github.com/kubernetes-client/rust) and populate it with the generated code.

**Pull request merged**